src/app/qgisapp.cpp

```diff
@@ -4464,6 +4464,8 @@ void QgisApp::addGeonodeLayer()
   }
   connect( geonodes, SIGNAL( addRasterLayer( QString const &, QString const &, QString const & ) ),
            this, SLOT( addRasterLayer( QString const &, QString const &, QString const & ) ) );
+  connect( geonodes, SIGNAL( addWfsLayer( QString, QString, QString ) ),
```

Comment:
This an old style connect. You need to use the new-style, QGIS doesn't allow new "old" style anymore.